package.json

```diff
@@ -20,16 +20,26 @@
       "import": "./js/tokens.js",
       "require": "./js/tokens.cjs"
     },
-    "./values.js": {
-      "import": "./js/values.js",
-      "require": "./js/values.cjs"
-    },
     "./json/*": {
       "require": "./json/*"
     },
     "./plugins/11ty.cjs": "./plugins/11ty.cjs",
     "./plugins/stylelint.cjs": "./plugins/stylelint.cjs",
-    "./css/*": "./css/*"
+    "./css/*": "./css/*",
+    "./values.js": {
+      "import": "./js/values.js",
+      "require": "./js/values.cjs"
```

Comment:
The file `./js/values.cjs` is no longer getting generated.  

I assume we should be creating this much in the way of ES modules *.js  but with *.cjs in `javascript/modules` rather than just adding the following block back to `platforms.yml`

```yaml
    - destination: values.cjs
      format: javascript/module-flat
      options:
        fileHeader: redhat/legal
```

I'll try and see if I can't get a commit in that does that very thing.

Reply:
I'm not so fussed over cjs. It can be 1.1